packages/website/components/types.d.ts

```diff
@@ -19,3 +19,7 @@ export type PageProps = RestrictedRouteProps &
   MetadataProps & {
     authOnLoad?: boolean;
   };
+
+export type PageAccountProps = PageProps & {
```

Comment:
@cmunns 
fwiw I think you can also define these in the js files via jsdoc like:

```
/**
 * @typedef {object} PageAccountProps
 * @property {string} [stripeKey]
 */
```

afaict either way is fine. just fyi